### ISSUE TITLE
CoolProp import doesn't work in v6.4.0

### ISSUE THREAD

**Issue by jfreissmann** [CoolProp/CoolProp#1940]
### Description

After upgrading to CoolProp version 6.4.0 the import does not work in python.

### Steps to Reproduce

1. Create empty virtual environment with python virtualenv
2. Install CoolProp with pip
3. Error raised while trying to import CoolProp

### Versions

**CoolProp Version:** 6.4.0
**Operating System and Version:** Windows 10
**Access Method:** Python 3.7.7

### Additional Information

Error message received:

``` python
import CoolProp
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\jfreissmann\test2_env\lib\site-packages\CoolProp\__init__.py", line 16, in <module>
    from .CoolProp import AbstractState
ImportError: DLL load failed: The named module was not found.
```

**Comment by ibell** [comment]
There have been scattered examples of this failure with 6.4.0.  It can be
fixed by installing the c++ redistributable have we are working to find a
solution that does not require that.

How did you get your Python?

**Comment by juliusmeier** [comment]
I have the same issue.
I have installed the Microsoft Visual C++ 2017 Redistributable (as well as the older versions: 2005, 2008, 2010, 2012, 2013) and Microsoft Visual C++ Build Tools 14.0

## Versions
CoolProp Version: 6.4.0
Operating System and Version: Windows 10
Access Method: Python 3.7.6 

I use Python through Anaconda

**Comment by jowr** [comment]
From the logs I can see that the extension was built using the 2019 version of MSVC - maybe you should try installing the latest C++ Redistributable as well.

**Comment by juliusmeier** [comment]
You are right! I installed the latest C++ Redistributable (in my case I was missing 2019) and now everything works well.
Thank you!

**Comment by ibell** [comment]
Sorry folks, this was my bad with the 6.4.0 release.  I will be shortly pushing an updated 6.4.1 release with the correct runtime dependency.  In the meantime, installation of the Microsoft C++ redistributable should do it.

**Comment by ibell** [comment]
After yet more digging, the offending DLL seems to be VCRUNTIME140_1.dll which Anaconda is not providing in the distribution. More info here: https://devblogs.microsoft.com/cppblog/making-cpp-exception-handling-smaller-x64/ .

**Comment by ibell** [comment]
Install the redistributable from here: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

**Comment by jfreissmann** [comment]
First of all, thank you for the quick responses.

Indeed, installing the latest [C++ Redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) through the link from @ibell fixed the issue for me.

**Comment by ibell** [comment]
Also, a solution is to do:
```
pip install msvc_runtime
```

But I am trying to get the package to auto-install without human intervention.

**Comment by ibell** [comment]
I think this is fixed now. Can someone with a fresh machine test it?  The msvc_runtime package will be forced to be installed, and that should fix this.

**Comment by AlicateDexter** [comment]
I tested here, it doesn't got automatically installed, but after I run  `pip install msvc_runtime` the problem was fixed.

**Comment by jowr** [comment]
This will be fixed with the next release v6.4.1 - v6.4.0 is known to be broken.
